src/main/java/com/hotels/beeju/ThriftHiveMetaStoreApp.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (C) 2015-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.beeju;
+
+import java.util.concurrent.CountDownLatch;
+
+public class ThriftHiveMetaStoreApp {
+
+  public static void main(String[] args) throws Throwable {
+    ThriftHiveMetaStoreJUnitRule rule = new ThriftHiveMetaStoreJUnitRule();
+    rule.setThriftPort(22334);
+    rule.before();
+    System.out.println("BeeJU Thrift Hive Metastore listening on: " + rule.getThriftConnectionUri());
```

Comment:
log?

Reply:
omg

Reply:
I was waiting for this. I did this on purpose, if I use slf4j it outputs nothing as it finds no appenders or config on the classpath. I could force the issue by adding a logging config under src/main/resources but this will then end up in the jar file, which we don't want. So I'd then need to override the jar plugin to exclude the logging file etc. I figured for now since this is only run in an IDE we can use sysout. If we ever decide to also ship a BeeJU executable jar which would allow you to run this "app" directly from the CLI we could do the whole logging thing.

Reply:
Beeju is a test library that can you pull into an existing project, that project needs to configure the logging. You don't specify a logging conf in src/main/resources leave it up to the "using" project.
If you want to print something out in BeeJU tests, add the necessary log impl as a "test" dep and add the logging conf in `src/test/resources`

Reply:
ah right it's an app with a main I missed that bit. Yeah sysout is better than. Not sure how useful this is though but ok, let's go for it :)

Reply:
That's the spirit!